azkaban/remote.py

```diff
@@ -701,11 +701,19 @@ def _run_options(self, name, flow, jobs=None, disabled_jobs=None,
       }[on_failure]
     except KeyError:
       raise ValueError('Invalid `on_failure` value: %r.' % (on_failure, ))
+    pipeline_level = 1
+    queue_level = 1
     if isinstance(concurrent, bool):
       concurrent = 'concurrent' if concurrent else 'skip'
+    elif 'pipeline-' in concurrent:
```

Comment:
Let's consolidate those two cases (and change the `-` separator to `:`):
```python
concurrency_level = None
if isinstance(concurrent, bool):
  concurrent = 'concurrent' if concurrent else 'skip'
elif ':' in concurrent:
  concurrent, concurrency_level = concurrent.split(':', 1)
```
We can then add the level to `request_data` as follows:
```python
request_data = {
  # ...
}
if concurrency_level is not None:
  request_data['%sLevel' % (concurrent, )] = concurrency_level
```